newsfragments/140.feature

```diff
@@ -0,0 +1 @@
+persist LocalSnapshots on the disk to preserve history even while the computer if offline.
```

Comment:
I like Twisted's style guidelines for news fragments: https://twistedmatrix.com/trac/wiki/ReviewProcess#Newsfiles

More generally, I guess we should have *some* style guide because the fragments are all assembled at release time and if they are consistent the result looks vastly better than if they are not.

(just wrote that on https://github.com/LeastAuthority/magic-folder/issues/164)

Reply:
Thanks. I wasn't aware how the news file was being used. I will submit a new PR that fixes the news file.